src/makeRobot.js

```diff
@@ -39,6 +39,56 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, `
+      + `wheels: ${wheels}`;
```

Comment:
you can do the same with wheels

Reply:
agree, don't use variable, use property of object
```suggestion
      + `wheels: ${this.wheels}`;
```